apps/meteor/server/models/raw/LivechatVisitors.ts

```diff
@@ -31,6 +31,7 @@ export class LivechatVisitorsRaw extends BaseRaw<ILivechatVisitor> implements IL
 			{ key: { name: 1 }, sparse: true },
 			{ key: { username: 1 } },
 			{ key: { 'contactMananger.username': 1 }, sparse: true },
+			{ key: { 'livechatData.$**': 1 } },
```

Comment:
If you're creating the index here, remove the one on the base model 👀

Reply:
ok!

Reply:
wait, do you mean the room model? it's a different collection than the visitors.

Reply:
No, I meant the other LivechatVisitors model you changed 👀 it's better to keep the indexes only on this one